Riot/Modules/Common/Recents/Views/RecentTableViewCell.m

```diff
@@ -92,12 +92,17 @@ - (void)render:(MXKCellData *)cellData
         self.lastEventDecriptionLabelTrailingConstraint.constant = self.unsentImageView.hidden ? 10 : 30;
 
         // Notify unreads and bing
-        if (roomCellData.hasUnread)
+        if (roomCellData.isRoomMarkedAsUnread)
         {
-
+            self.missedNotifAndUnreadBadgeBgView.hidden = NO;
+            self.missedNotifAndUnreadBadgeBgView.backgroundColor = ThemeService.shared.theme.tintColor;
+            self.missedNotifAndUnreadBadgeBgViewWidthConstraint.constant = 20;
```

Comment:
This probably has nothing to do with your PR but why do we even need to change the constraint? Isn't hiding the view enough? 🤔

Reply:
well since that constraint is modified elsewhere, I just give the default value to make view rounded
`self.missedNotifAndUnreadBadgeBgViewWidthConstraint.constant = self.missedNotifAndUnreadBadgeLabel.frame.size.width + 18;`